src/Console/Command/Local.php

```diff
@@ -8,19 +8,41 @@
 use Exception;
 use JsonException;
 use Psr\Container\NotFoundExceptionInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Yaml\Yaml;
 use Throwable;
 
 /**
  * Local function invocation.
  */
-class Local
+class Local extends Command
 {
+    protected static $defaultName = 'local';
+
     public const SIGNATURE = 'local [function] [data] [--file=] [--handler=] [--config=]';
```

Comment:
This const is not used.

Reply:
Strictly speaking this is a BC break but should be ok anyway.

Reply:
Removing `__invoke` from this class is a more risked breaking change. I don't know what's the policy here.

Reply:
Feel free to remove, no BC break guarantee for console commands.